pkg/network/config/config_test.go

```diff
@@ -1463,34 +1463,34 @@ func TestUSMTLSNativeEnabled(t *testing.T) {
 func TestUSMTLSGoEnabled(t *testing.T) {
 	t.Run("via deprecated YAML", func(t *testing.T) {
 		mockSystemProbe := mock.NewSystemProbe(t)
-		mockSystemProbe.SetWithoutSource("service_monitoring_config.enable_go_tls_support", true)
+		mockSystemProbe.SetWithoutSource("service_monitoring_config.enable_go_tls_support", false)
 		cfg := New()
 
-		require.True(t, cfg.EnableGoTLSSupport)
+		require.False(t, cfg.EnableGoTLSSupport)
 	})
 
 	t.Run("via deprecated ENV variable", func(t *testing.T) {
 		mock.NewSystemProbe(t)
-		t.Setenv("DD_SERVICE_MONITORING_CONFIG_ENABLE_GO_TLS_SUPPORT", "true")
+		t.Setenv("DD_SERVICE_MONITORING_CONFIG_ENABLE_GO_TLS_SUPPORT", "false")
 		cfg := New()
 
-		require.True(t, cfg.EnableGoTLSSupport)
+		require.False(t, cfg.EnableGoTLSSupport)
 	})
 
 	t.Run("via YAML", func(t *testing.T) {
 		mockSystemProbe := mock.NewSystemProbe(t)
-		mockSystemProbe.SetWithoutSource("service_monitoring_config.tls.go.enabled", true)
+		mockSystemProbe.SetWithoutSource("service_monitoring_config.tls.go.enabled", false)
 		cfg := New()
 
-		require.True(t, cfg.EnableGoTLSSupport)
+		require.False(t, cfg.EnableGoTLSSupport)
 	})
 
 	t.Run("via ENV variable", func(t *testing.T) {
 		mock.NewSystemProbe(t)
-		t.Setenv("DD_SERVICE_MONITORING_CONFIG_TLS_GO_ENABLED", "true")
+		t.Setenv("DD_SERVICE_MONITORING_CONFIG_TLS_GO_ENABLED", "false")
 		cfg := New()
 
-		require.True(t, cfg.EnableGoTLSSupport)
+		require.False(t, cfg.EnableGoTLSSupport)
```

Comment:
Instead of changing the tests, I'd add a scenario for `true` and `false`

Reply:
I am not sure I get it. I changed the tests because, to me, their value is in proving that setting the config via yaml or env works. If I leave them unchanged, the tests no longer prove it works because we can't differentiate if the config value is true because of the yaml/env or because of the default value.

Reply:
So, I noticed now that making the change of the default value in `adjust_usm`, alongside the ones for HTTP, HTTP2, etc..., does not require actually changing the tests at all. Although, it means having a test for a "default value" that checks it is false (which is true only if USM is not enabled), like we have for HTTP config.

Reply:
So, it turns out that with the change made in `adjust_usm.go`, the tests are no longer affected (they still have a false default value, like we have for other protocols like the HTTP config tests). So I reverted the changes to them.